pieces.py

```diff
@@ -0,0 +1,336 @@
+    BLACK_IMAGE = pygame.image.load(os.path.join(BLACK_PIECES_PATH, 'black_pawn.png'))
+    SCORE_EVOLUTION_TABLE = [[0,   0,   0,   0,   0,   0,  0,   0],
+                             [50, 50,  50,  50,  50,  50, 50,  50],
+                             [10, 10,  20,  30,  30,  20, 10,  10],
+                             [5,   5,  10,  25,  25,  10,  5,   5],
+                             [0,   0,   0,  20,  20,   0,  0,   0],
+                             [5,  -5, -10,   0,   0, -10, -5,   5],
+                             [5,  10,  10, -20, -20,  10, 10,   5],
+                             [0,   0,   0,   0,   0,   0,  0,   0]]
+
+    def __init__(self, team: Team, tur=None, square=None):
+
+        if square is None:
+            square = squares[Pawn.LINE_OF_WHITE_PAWNS][tur] if team.is_white_team else squares[Pawn.LINE_OF_BLACK_PAWNS][tur]
+            super().__init__(square, team)
+        else:
+            super().__init__(square, team)
+
+    def is_reached_to_end(self):
+        end = BOARD_LINE-1 if self.team.is_white_team else 0
+        return self.square.line_cord == end
+
+    def get_valid_move_squares(self):
+        line = self.square.line_cord
+        tur = self.square.tur_cord
+        valid_moves = []
+        # Direction represent one square walk.
+        direction = 1 if self.team.is_white_team else -1
+        line += direction
+        if line > 7 or line < 0:
+            return valid_moves
+
+        # Check if nest step is out of board.
+        next_square = squares[line][tur]
+        valid_moves.extend(self._diagonal_eat(next_square))
+
+        if next_square.current_piece is not None:
+            return valid_moves
+
+        valid_moves.append(next_square)
+        if self.move_counter == 0:
+            line += direction
+            next_square = squares[line][tur]
+            if next_square.current_piece is None:
+                valid_moves.append(next_square)
+
+        return valid_moves
+
+    def _diagonal_eat(self, next_square):
+        valid_eat_moves = []
+
+        # Left check.
+        line = next_square.line_cord
+        tur = next_square.tur_cord - 1
+        if is_move_to_square_valid(tur, line, self.team):
+            current_square = squares[line][tur]
+            if current_square.current_piece is not None:
+                valid_eat_moves.append(current_square)
+        # Right check.
+        tur += 2
+        if is_move_to_square_valid(tur, line, self.team):
+            current_square = squares[line][tur]
+            if current_square.current_piece is not None:
+                valid_eat_moves.append(current_square)
+        return valid_eat_moves
+
+
+class Knight(Piece):
+    BASIC_SCORE = 320
+    BLACK_IMAGE = pygame.image.load(os.path.join(BLACK_PIECES_PATH, 'black_knight.png'))
+    WHITE_IMAGE = pygame.image.load(os.path.join(WHITE_PIECES_PATH, 'white_knight.png'))
+    SCORE_EVOLUTION_TABLE = [[-50,-40,-30,-30,-30,-30,-40,-50],
```

Comment:
Please make the SCORE_EVOLUTION_TABLE to look more organized as the other pieces tables.

Reply:
fixed